### PR TITLE
[sr_Latn_ME] Translation fix.

### PR DESCRIPTION
**Pull request by milose**
Translated and fixed most of Montenegrin language strings.

### CODE REVIEW

File: src/sr_Latn_ME/pagination.php [hunk 1]
```diff
@@ -12,6 +12,6 @@
     |
     */
 
-    'previous' => '&laquo; Prethodna',
-    'next'     => 'Sledeća &raquo;',
+    'previous' => '&laquo; prethodna',
+    'next' => 'sljedeća &raquo;',
```
Comment (caouecs) [inline src/sr_Latn_ME/pagination.php:16]: Don't change indentation

**Review by caouecs — CHANGES_REQUESTED** [review]
thank you, just a problem on indentation

File: src/sr_Latn_ME/pagination.php (hunk 1 above)
Comment (milose) [reply src/sr_Latn_ME/pagination.php:16]: Sorry about that.

**Comment by caouecs** [comment]
Thank you

**Pull request merged**